src/Plugin/RulesDataMatcher/MixedGreaterThanMatcher.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\RulesDataMatcher\MixedGreaterThanMatcher.
+ */
+
+namespace Drupal\rules\Plugin\RulesDataMatcher;
+
+use Drupal\rules\Context\DataMatcherInterface;
+
+/**
+ * Defines a 'greater than' matcher.
+ *
+ * @RulesDataMatcher(
+ *   id = "rules_mixed_greater_than",
+ *   label = @Translation("A 'greater than' matcher."),
+ *   subject_type = "mixed",
```

Comment:
Instead of mixed, we should use any as well. But  any would allow for objects and lists as well, so I guess we want to have "float" here? integers are usable as float also. But then, what about comparing dates? Would it make sense to support an array of types here also?